### ISSUE TITLE
Docker, Alpine(, and maybe Flatpak)

### ISSUE THREAD

**Issue by oemsysadm** [openbao/openbao#327]
**Is your feature request related to a problem? Please describe.**
Ease of installation in dev and container environments.

**Describe the solution you'd like**
I'd like docker images, ideally via Alpine Linux repository support (to mirror OpenTofu). And since it's an always running application, for development/test purposes it would be nice to have flathub.org as an installation option.

**Comment by JanMa** [comment]
Hey @oemsysadm ,
we already build a docker image for x86 and one for arm. For now you can download them from the release assets of our latest release: https://github.com/openbao/openbao/releases/tag/v2.0.0-alpha20240329

To import the image and run it locally, please run the following command:

```shellscript
curl -sL "https://github.com/openbao/openbao/releases/download/v2.0.0-alpha20240329/openbao_ubi_linux_amd64_2.0.0-alpha20240329_f20d4684d48c8f841025a50c3cb54aa12434e8e0.docker.tar.zip" -o openbao.tar;
unzip -p openbao.tar | docker load
```

We already have a docker hub account, but we're still waiting on it to be recognized as an open source project so the traffic limits are lifted. Once that happened, we will also upload our images there as one would expect
